src/evaluators/ClassDeclaration.js

```diff
@@ -10,11 +10,272 @@
 /* @flow */
 
 import type { Realm } from "../realm.js";
-import type { LexicalEnvironment } from "../environment.js";
+import type { LexicalEnvironment, Reference } from "../environment.js";
 import type { Value } from "../values/index.js";
-import type { Reference } from "../environment.js";
-import type { BabelNodeClassDeclaration } from "babel-types";
+import { NullValue, EmptyValue, ObjectValue } from '../values/index.js';
+import type { BabelNodeClassDeclaration, BabelNodeExpression, BabelNodeClassMethod } from "babel-types";
+import parse from "../utils/parse.js";
+import {
+  HasOwnProperty,
+  SetFunctionName,
+  IsConstructor,
+  Get,
+  MakeConstructor,
+  CreateMethodProperty,
+  DefineMethod,
+  NewDeclarativeEnvironment,
+  MakeClassConstructor,
+  ObjectCreate,
+  ConstructorMethod,
+  GetValue,
+  IsStatic,
+  PropertyDefinitionEvaluation,
+  InitializeBoundName,
+  NonConstructorMethodDefinitions,
+} from "../methods/index.js";
+import { AbruptCompletion } from "../completions.js";
+import invariant from "../invariant.js";
 
+function EvaluateClassHeritage(realm: Realm, ClassHeritage: BabelNodeExpression, strictCode: boolean): ObjectValue | null {
```

Comment:
There's no spec section, I just pulled it out as a convenience method because I was having trouble getting the evaluation of ClassHeritage to return the right thing. Thought I saw a similar pattern elsewhere in the codebase but I can't seem to find it now. Should I move it back inline?